.appveyor.yml

```diff
@@ -5,7 +5,7 @@ clone_folder: c:\projects\symfony
 init:
     - SET PATH=c:\php;%PATH%
     - SET COMPOSER_NO_INTERACTION=1
-    - SET SYMFONY_DEPRECATIONS_HELPER=strict
+    - SET "SYMFONY_DEPRECATIONS_HELPER=max[total]=0&baselineFile=c:\projects\symfony\.github\deprecations-baseline.json"
```

Comment:
strict === max[total]=0
It looks like we cannot use a predefined mode such as strict or weak AND another option